Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs

```diff
@@ -102,7 +102,7 @@ public void Unarchive(CancellationToken cancellationToken)
             DebugLogger.Log("Unpacking finished succesfully!");
         }
 
-        public void UnarchiveSingleFile(Pack1Meta.FileEntry file, CancellationToken cancellationToken)
+        public void UnarchiveSingleFile(Pack1Meta.FileEntry file, CancellationToken cancellationToken, string destinationDirPath = null)
```

Comment:
No, it shouldn't be a directory path, but a file name (file path). Why a directory?

Reply:
Well we have a file name in the FileEntry, do we want ovveride that as well?